### DISCUSSION TITLE
DisplayPromptAsync and landscape mode

### DISCUSSION THREAD

**Discussion by sk479** [CommunityToolkit/Maui#1978] [Q&A]
We have a formerly Xamarin and now MAUI application that requires user input via the soft keyboard.
When the application is running in landscape mode on Android, the user input prompt is hidden by the soft keyboard which requires the user to enter their desired text, press Done on the keyboard, then press OK at the display prompt.
On iOS this does not occur and the keyboard appears under the display prompt window as expected.
What can be done to prevent the soft keyboard from hiding the display prompt window?

**Comment by pictos** [comment]
Hey @sk479 this isn't the .net maui repo. I recommend you to open this discussion [here](https://github.com/dotnet/maui/discussions/)
